From 29c908647eec8e05674ba1c298d4f1c565d9f872 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Corentin=20No=C3=ABl?= <corentin@elementary.io>
Date: Sun, 26 Apr 2020 11:54:46 +0200
Subject: [PATCH] pxgsettings: use the correct syntax to connect to the changed
 signal

As it is a detailed signal, it only makes sense to append a :: when there is a specific property to target.
It used to be accepted but triggers a runtime warning with latest GLib.
---
 libproxy/modules/pxgsettings.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libproxy/modules/pxgsettings.cpp b/libproxy/modules/pxgsettings.cpp
index 9ed4333..0db5a6a 100644
--- a/libproxy/modules/pxgsettings.cpp
+++ b/libproxy/modules/pxgsettings.cpp
@@ -158,7 +158,7 @@ int main(int argc, char **argv) {
 #else
 		gchar** keys = g_settings_list_keys(settings);
 #endif
-		g_signal_connect(settings, "changed::", G_CALLBACK (on_value_change), argv[i]);
+		g_signal_connect(settings, "changed", G_CALLBACK (on_value_change), argv[i]);
 		for (int j=0; keys[j]; on_value_change(settings, keys[j++],argv[i] ));
 		g_strfreev(keys);
 	}

